debezium-api/pom.xml

```diff
@@ -4,7 +4,7 @@
         <groupId>io.debezium</groupId>
         <artifactId>debezium-parent</artifactId>
         <version>2.1.0-SNAPSHOT</version>
-	<relativePath>../debezium-parent/pom.xml</relativePath>
+        <relativePath>../debezium-parent/pom.xml</relativePath>
```

Comment:
What happened here?

Reply:
@rk3rn3r tab vs spaces

Reply:
Oh, I think I tried to mark the text in front and it showed me whitespaces on both sides. Maybe I was somehow too fast... now I see tabs. Sorry! :hand_over_mouth: